docs/web/errors.mdx

```diff
@@ -141,6 +115,59 @@ Alternatively, `findDetails()` takes a registry as an argument. See the
 for details.
 
 
+
+## Working with errors
+
+The method of handling errors will depend on the type of client being used.
+
+### Promise Clients
+
+[Promise-based clients](./using-clients#promises) can use a try-catch block to catch the error. Note that you must
+verify it is a `ConnectError` first.
+
+```ts
+import { ConnectError } from "@connectrpc/connect";
+
+try {
+  await client.say({sentence: ""});
+} catch (err) {
+  // We have to verify err is a ConnectError
+  // before using it as one.
+  if (err instanceof ConnectError) {
+    err.code;    // Code.InvalidArgument
+    err.message; // "[invalid_argument] sentence cannot be empty"
+  }
+  // Alternatively, we can use ConnectError.from()
+  // It returns a ConnectError as is, and converts any
+  // other error to a ConnectError.
+  const connectErr = ConnectError.from(err);
+  connectErr.code;    // Code.InvalidArgument
+  connectErr.message; // "[invalid_argument] sentence cannot be empty"
+}
+```
+
+### Callback Clients
```

Comment:
I don't think we should show code examples for both clients shapes here. There are _many_ other places where we would need to do it as well for consistency, and I am not convinced that the end result would be helpful. 

We use promises exclusively in the documentation, and only surface callback clients on the "Using clients" page. If you think that it is not clear from "Using clients" how to catch errors, let's document it there.